packages/core/echo/echo-signals/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@dxos/echo-signals",
```

Comment:
rename to @dxos/signals-preact as its preact-specific?

Reply:
Also this might be an `sdk` level package

Reply:
It's not preact-specific, in fact, this is currently _react_ specific, but my plan was to include different entrypoints like `deepsignal` does. I'd be fine with this being an sdk package, though I figured it belonged here because it's an implementation of an api exposed by echo-schema